string filename
---
int16 result

